mimoto-issuers-config.json

```diff
@@ -325,10 +330,11 @@
           "token_endpoint": "https://${mosip.api.public.host}/v1/mimoto/get-token/Mock-Fallback",
           "proxy_token_endpoint": "https://${mosip.esignet.mock.host}/v1/esignet/oauth/v2/token",
           "qr_code_type": "OnlineSharing",
+          "credential_issuer": "http://config-server.config-server1",
```

Comment:
Is this the issuer?

Reply:
Not a issuer, it is for fallback scenario just copied from exisitng wellknown base url. Shall we remove all fallback issuers?

Reply:
Removed fallback issuers from list.